src/libraries/Native/Unix/System.Security.Cryptography.Native/apibridge.c

```diff
@@ -479,6 +479,16 @@ int32_t local_RSA_set0_crt_params(RSA* rsa, BIGNUM* dmp1, BIGNUM* dmq1, BIGNUM*
     return 1;
 }
 
+int32_t local_SSL_CTX_config(SSL_CTX* ctx, const char* name)
```

Comment:
Where is this used? Is there part missing reading `name` from ENV or app context?

Reply:
If portable binding targets OpenSSL 1.0.x it calls this method, since SSL_CTX_config wasn't defined yet.  The only call passes in "system_default", because that's the way that you ask for the system default configuration in 1.1.x.